user/urls.py

```diff
@@ -1,5 +1,11 @@
+"""Urls.py manages the urls for the user management
```

Comment:
Linebreak, remove "Urls.py" because it is clear that this comment is about what urls.py does